storage-operator/deploy/crds/storage_v1alpha1_volume_crd.yaml

```diff
@@ -78,28 +78,42 @@ spec:
           type: object
         status:
           properties:
-            errorCode:
-              description: Volume failure error code
-              enum:
-              - InternalError
-              - CreationError
-              - DestructionError
-              - UnavailableError
-              type: string
-            errorMessage:
-              description: Volume failure error message
-              type: string
+            conditions:
+              description: List of conditions through which the Volume has or has
+                not passed.
+              items:
+                properties:
+                  lastTransitionTime:
+                    description: Last time the condition transit from one status to
```

Comment:
```suggestion
                    description: Last time the condition transited from one status to
```

Reply:
This file is auto generated, but I'll fix the comment from which this is generated.
Done